docker/Dockerfile_ci

```diff
@@ -1,84 +1,113 @@
-FROM anapsix/alpine-java:8_jdk
+FROM ubuntu:latest
 
 LABEL maintainer="amann@st.informatik.tu-darmstadt.de"
 
-# Install ca-certificates so that HTTPS works consistently
-RUN apk update \
- && apk add ca-certificates wget \
- && update-ca-certificates
-
+# settings
+ENV DEBIAN_FRONTEND noninteractive
 ENV LANG=C.UTF-8 \
     GRADLE_VERSION=4.0.2 \
     GRADLE_HOME=/usr/local/gradle
-ENV PATH=${PATH}:${GRADLE_HOME}/bin
+
+RUN apt-get clean \
+      && apt-get update \
+      && apt-get install -y \
+            software-properties-common \
+            locales \
+            ca-certificates \
+            unzip \
+      && update-ca-certificates \
+      && locale-gen en_US en_US.UTF-8 \
+      && dpkg-reconfigure locales \
+      && echo "APT::Get::Assume-Yes \"true\";\nAPT::Get::force-yes \"true\";" >> /etc/apt/apt.conf.d/90forceyes
+
+# Install java8
+RUN add-apt-repository ppa:webupd8team/java \
+      && apt-get update \
+      && echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | debconf-set-selections \
+      && apt-get install -y oracle-java8-installer
+
+ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
+ENV PATH $PATH:$JAVA_HOME/bin
+
+RUN update-alternatives --set java "$JAVA_HOME/jre/bin/java" \
```

Comment:
You should be able to set the ENV before installing Java, which allows you to merge the two RUNs into one and make the image smaller.